src/home/home.ts

```diff
@@ -86,6 +90,17 @@ window.home = (): void => {
         document.getElementById('reading-pane-popup').innerHTML = value.toString();
         window.readingpane();
     });
+    let appObject: AppBar = new AppBar({
+        colorMode: 'Dark'
+    })
+    appObject.appendTo("#appbar");
+    defaultSidebar = new Sidebar({
+        width: "280px",
+        type: "Push",
```

Comment:
Use the sidebar backdrop and remove the overlay element added manually

Reply:
Used the sidebar backdrop .